webpack/components/AnsibleRolesSwitcher/components/AssignedRolesList.js

```diff
@@ -31,40 +77,25 @@ const AssignedRolesList = ({
             dropdownButtonId="assigned-ansible-roles-pagination-row-dropdown"
           />
         </div>
-        {assignedRoles.map(role => (
-          <AnsibleRole
+        {assignedRoles.map(renderRole)}
```

Comment:
shouldn't we order the inherited roles first? :thinking:
They will come first, so we should make sure we display it that way, right?
What about sorting the roles beforehand like (expects we have the position of the roles):
```js
assignedRoles.sort((r1, r2) => { if (r1.inherited && r2.inherited) return 0; if (r1.inherited) return -1; if (r2.inherited) return 1; return r1.position - r2.position; };
```

Reply:
We do not need to order here, roles already have correct order.